app/views/creature_actions/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<p id="notice"><%= notice %></p>
+
```

Comment:
There needs to be stuff here describing a single action. Its name, description, damage dice, range, and which creature it belongs to.